assets/Product.wxs

```diff
@@ -165,6 +166,10 @@
             <Component Id="Telemetry" Guid="{80520f20-471d-4d08-adc8-bab637627b39}" KeyPath="yes">
               <Environment Id="PowerShellDistributionChannel" Action="create" Name="POWERSHELL_DISTRIBUTION_CHANNEL" Permanent="no" System="yes" Value="MSI:[WINDOWS_PRODUCT_NAME]"/>
             </Component>
+            <Component Id="DisableTelemetry" Guid="{E86F5C91-3721-4C5C-A31F-1F0A5A468573}" KeyPath="yes">
```

Comment:
Yes, the GUID here is necessary because there's no possible key path, so WiX has nothing from which to calculate a durable GUID.

Reply:
Thanks for confirming :-)